bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOIntegrationTest.java

```diff
@@ -102,44 +116,78 @@ public void testEstimatedAndSplitForSmallTable() throws Exception {
   public void testEstimatedAndSplitForLargeTable() throws Exception {
```

Comment:
The change of this PR is caching SampleRowKeysResponse, can you please verify this change by doing following:

1. getEstimatedSizeBytes()
2.  SampleRowKeys = GetSampleRowKeys();
3. splitIntoBundles()
4. assert same (SampleRowKeys, GetSampleRowKeys())

or have a separate test method that calls GetSampleRowKeys() twice to verify the same SampleRowKeys.